data/Amrendra_Singh.json

```
@@ -0,0 +1,34 @@
+{
+  "name": "Amrendra Snigh",
```

```suggestion
  "name": "Amrendra Singh",
```
Maybe name was misspelled